### PR TITLE
Add scripts of downloading Cryosat-2 ice observations

### LINKED ISSUE #174: Add scripts for downloading ice obs
Create bash scripts for downloading Cryosat-2 ice obs from ESA.

### PR DESCRIPTION
**Pull request by YanHao-NOAA**
Scripts "source.esa_ice_cryosat.sh" was created to download Cryosat-2 ice observations from ESA ftp site.
In get_obs.sh, only Cryosat GDR (LRM+SAR+SIN) is used. If users want to use LRM, SAR, SIN data separately, those bash lines with LRM, SAR or SIN need to be uncommented.  
Please close #174

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
looks good.
Thanks for adding this source.

File: scripts/obs/get_obs.sh [hunk 1]
```diff
@@ -40,6 +40,10 @@ while [[ $date -le $date_end ]]; do
     ## ice concentration
     ##------------------------------------------------------------
     bash source.nsidc_icec.sh $date $output_path
+    bash source.esa_ice_cryosat.sh GDR $date $output_path
```
Comment (guillaumevernieres) [inline scripts/obs/get_obs.sh:47]: cryosat is not ice concentration, it needs it's own section ## ice freeboar/thickness

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
Thanks for doing that, just a minor comment to address.

File: scripts/obs/get_obs.sh (hunk 1 above)
Comment (YanHao-NOAA) [reply scripts/obs/get_obs.sh:47]: Excellent point! Thank you! I changed get_obs.sh to adding section ## ice freeboard/thickness.
But git push can not connect from my home (I tried a few times, this work is on Hera). I will fix this on Monday from my office.

**Review by guillaumevernieres — APPROVED** [review]
:+1:

**Comment by flampouris** [comment]
The ice script needs a small update, Yan will push it on Monday.

**Pull request merged**